docs/getting-started/part-1/get-departures.md

```diff
@@ -157,8 +164,8 @@ podman run --rm -v ${PWD}/data:/mnt/data -v ${PWD}/target:/mnt/lib -v ${PWD}/con
 After executing it, you will see the file *data/stg_departures/result.json* has been replaced with the output of your pipeline.
 
 :::caution
-Since both web servers are freely available on the internet, they might be overloaded by traffic.
-If the download fails because of a timeout, wait a couple of minutes and try again.
+Since both web servers are freely available on the internet, **a rate limiting applies**. https://opensky-network.org/ will stop responding if you make too many calls.
+If the download fails because of a timeout, wait a couple of minutes and try again. In the worst case, it should work again on the next day.
```

Comment:
Sounds a bit strange IMHO, "in the worst case, it should work again".
```suggestion
If the download fails because of a timeout, wait a couple of minutes and try again. In the worst case, it will only work again the following day.
```